### PR TITLE
Add Switzerland to the countries list & hint text to the country page

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* Add hint text to the country page
* Add the missing dependencies hof-controller, lodash, express, winston
* Replace hmpo-model with hof-model
* Updated hof-bootstrap to 9.0.0
* Replaced node-sass with npm-sass

https://jira.digital.homeoffice.gov.uk/browse/GFM-201

### CODE REVIEW

File: assets/countries/index.js [hunk 1]
```diff
@@ -230,6 +230,7 @@ var countriesNonEU = [
   'Suriname',
   'Svalbard and Jan Mayen Islands',
   'Swaziland',
```
Comment (sulthan-ahmed) [inline assets/countries/index.js]: Yes Swaziland

File: package.json [hunk 2]
```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
+    "express": "^4.14.0",
     "hof-bootstrap": "^6.0.0",
```
Comment (JoeChapman) [inline package.json]: Please update to `^9.0.0`

Comment (lennym) [inline package.json]: Yes, but not in scope for this PR.

File: package.json [hunk 3]
```diff
@@ -34,12 +34,17 @@
...
+    "hof-controllers": "^4.0.1",
     "hof-frontend-toolkit": "^1.0.0",
+    "hof-model": "^2.1.0",
     "hof-transpiler": "^0.1.0",
     "jquery": "^3.1.0",
+    "lodash": "^4.16.3",
     "node-sass": "^3.10.0",
```
Comment (JoeChapman) [inline package.json]: I overheard some chat about using `npm-sass` instead.

Comment (lennym) [inline package.json]: Yes, but not in scope for this PR.

File: package.json (hunk 2 above)
Comment (JoeChapman) [inline package.json]: ok, are the other dependency version bumps in the scope of this PR?

Comment (lennym) [inline package.json]: There are none. There are only adding dependencies that were previously undefined, I assume to satisfy the eslint rules.

File: package.json (hunk 3 above)
Comment (JoeChapman) [inline package.json]: That is fine, I wouldn't have mentioned it but there seem to be some deps updated.

File: package.json [hunk 4]
```diff
@@ -34,12 +34,17 @@
   "author": "HomeOffice",
   "dependencies": {
     "browserify": "^12.0.1",
-    "hof-bootstrap": "^6.0.0",
-    "hof-frontend-toolkit": "^1.0.0",
+    "express": "^4.14.0",
```
Comment (joefitter) [inline package.json]: Do we need express? If so should it be a dev dep?

File: package.json [hunk 5]
```diff
@@ -34,12 +34,17 @@
...
+    "hof-frontend-toolkit": "^1.0.1",
+    "hof-model": "^2.1.0",
     "hof-transpiler": "^0.1.0",
     "jquery": "^3.1.0",
-    "node-sass": "^3.10.0",
-    "typeahead-aria": "^1.0.1"
+    "lodash": "^4.16.3",
+    "npm-sass": "^2.0.0",
+    "typeahead-aria": "^1.0.1",
+    "winston": "^2.3.0"
```
Comment (joefitter) [inline package.json]: we need winston?

Comment (joefitter) [inline package.json]: remove the logger!

File: package.json (hunk 4 above)
Comment (joefitter) [inline package.json]: we _do_ need express

Comment (lennym) [inline package.json]: We need express, but only for dev-ing. So make it a dev-dependency.

Comment (sulthan-ahmed) [inline package.json]: That's what I did initially but my linting complained
```mock-postcode.js
  2:16  error  Module "express" is not listed as a dependency in package.json  implicit-dependencies/no-implicit```

Does this mean the linting is broken? Should I fix the linting? In the mean time should I put in a `es-lint ignore line`

Comment (lennym) [inline package.json]: There is nothing wrong with the linting. It (correctly) assumes everything in `./test` is a dev concern and everything else is a prod concern.

In this case, there is an exception, so you need to override it with manual config.

**Comment by JoeChapman** [comment]
Can you use this instead https://github.com/UKHomeOffice/homeoffice-countries?

**Comment by sulthan-ahmed** [comment]
yes @JoeChapman good idea I just recently learnt about this while working on right to rent.  I'll make the change

**Comment by sulthan-ahmed** [comment]
thank you @lennym

**Pull request merged**